config_sample/config.yaml

```diff
@@ -32,6 +32,9 @@ local: false
 # revoked.
 modpass: mod
 
+#default ban duration goes here
+bandur: 6
```

Comment:
There's no need to be ultra-concise in a configuration file - just say `default_ban_duration`.

No units are specified. There is no indication that the number 6 refers to 6 hours.

Use a string instead that accepts the same duration syntax as the `/ban` command (including `perma`).

Suggestion:
```yaml
# If no ban duration is specified, this one will be used.
# This duration follows the same syntax as the /ban command.
# Examples: "6 hours", "6h", "1 day", "1d", "perma"
default_ban_duration: 6h
```